prov/efa/src/rdm/rxr_ep.c

```diff
@@ -2197,6 +2197,17 @@ bool rxr_ep_use_shm_for_tx(struct fi_info *info)
 	    && !(info->caps & FI_LOCAL_COMM))
 		return 0;
 
+	/*
+	 * shm provider must make cuda calls to transfer cuda memory.
+	 * if cuda call is not allowed, we cannot use shm for transfer.
+	 * Note that, the other 2 hmem interfaces supported by EFA:
+	 * AWS Neuron and Habans synapse, will not pass intra-node traffic
+	 * to EFA (and shm provider does not support them either),
+	 * therefore disable shm will not impact them.
+	 */
+	if (info && (info->caps & FI_HMEM) && cuda_get_xfer_setting()!= CUDA_XFER_ENABLED)
```

Comment:
nit missing space `cuda_get_xfer_setting()!<space>= CUDA_XFER_ENABLED`

Reply:
fixed in new revision